code/game/objects/items/storage/boxes.dm

```diff
@@ -760,3 +760,211 @@
 	else if(!isopened)
 		isopened = 1
 		icon_state = "mealpackopened"
+
+//food boxes for storage in bulk
+
+//meat
+/obj/item/storage/box/meat
+	name = "\improper Meat Box"
+
+/obj/item/storage/box/meat/fill_preset_inventory()
+	for(var/i = 0, i < 7, i++)
+		new /obj/item/reagent_container/food/snacks/meat/monkey(src)
+
+//fish
+/obj/item/storage/box/fish
+	name = "\improper Fish Box"
+
+/obj/item/storage/box/fish/fill_preset_inventory()
+	for(var/i = 0, i < 7, i++)
+		new /obj/item/reagent_container/food/snacks/carpmeat(src)
+
+//grocery
+
+//milk
+/obj/item/storage/box/milk
+	name = "\improper Milk Box"
+
+/obj/item/storage/box/milk/fill_preset_inventory()
+	for(var/i = 0, i < 7, i++)
+		new /obj/item/reagent_container/food/drinks/milk(src)
+
+//soymilk
+/obj/item/storage/box/soymilk
+	name = "\improper Soymilk Box"
```

Comment:
why are these proper nouns and not formatted like “box of soymilk” like the above file

Reply:
replaced